src/actions/userActions.js

```diff
@@ -86,12 +87,21 @@ export function refreshUserInfo() {
 export function auth0Login(authResult) {
   return function(dispatch) {
     return new Promise(function(resolve) {
+      let isAdmin = false;
+      if (
+        authResult.idTokenPayload['https://giantswarm.io/groups'] ===
```

Comment:
It will contain only one group?
I was thinking of what would happen if there is more than one group. It could be stronger to check after splitting by space|comma (dont know how it will be separated) and then check if it exists.

Reply:
Yeah I like not adding code before we know what we'll do with it or if we'll need it... so for now wanted to do the 'dumbest' thing possible.